pages/workspace/workspace.html.js

```diff
@@ -206,11 +206,19 @@ function Workspace({ navTitle, showLogs, resLocals }) {
         <main class="d-flex flex-column flex-grow h-100">
           <div
             id="loading"
-            class="d-flex flex-grow h-100 justify-content-center align-items-center"
+            class="d-flex h-100 flex-grow justify-content-center align-items-center"
           >
             <i class="d-block fa fa-10x fa-circle-notch fa-spin text-info" aria-hidden="true"></i>
             <span class="sr-only">Loading workspace &hellip;</span>
           </div>
+          <div
+            id="stopped"
+            class="d-none h-100 flex-grow flex-column justify-content-center align-items-center"
+          >
+            <h2>Workspace stopped due to inactivity</h2>
+            <p>Your data was automatically saved. Reload the page to restart the workspace.</p>
```

Comment:
Does this copy sound ok? I kind of just threw it together. Here's how it looks:

<img width="990" alt="Screenshot 2023-02-23 at 17 15 43" src="https://user-images.githubusercontent.com/1476544/221304845-962c07fe-dd14-49fb-a22b-005063230840.png">

Reply:
Sounds good! Of course we are relying on the workspace to have actually automatically saved their data, but I think it's good to be reassuring about this.

Reply:
Yeah, I would consider any workspace that *didn't* automatically save data as fundamentally broken for our use case. Might be worth documenting that expectation? I don't think we have docs for building custom workspace images, FWIW.

Reply:
Good point, it would be great to explicitly document this somewhere in https://prairielearn.readthedocs.io/en/latest/workspaces/

Reply:
I documented this expectation in 743cdc6.